dvc/_debug.py

```diff
@@ -77,15 +77,15 @@ def add_debugging_flags(parser):
     from argparse import SUPPRESS
 
     parser.add_argument(
-        "--cprofile", action="store_true", default=False, help=SUPPRESS,
+        "--cprofile", action="store_true", default=False, help=SUPPRESS
```

Comment:
Starting from 19.3b0, black started to explode this into multiple lines. Therefore, I am removing those trailing commas for now.

Reply:
We should probably set `--skip-magic-trailing-comma` in the config. But that's my personal taste, I think.